IFC4x3/Properties/a/AssembleReference_1j3CDAXrnEIAdLgmnJ5ij5/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AssembleReference_1j3CDAXrnEIAdLgmnJ5ij5" Name="AssembleReference" UniqueId="6d0cc34a-875c-4e48-a9d5-ab0c5316cb45" PropertyType="p_referencevalue" />
```

Comment:
This property is missing a primary measure type (what would this be?).  Was the use of a Document Reference considered as an alternative to reference an external document? https://standards.buildingsmart.org/IFC/RELEASE/IFC4/ADD2_TC1/HTML/link/ifcdocumentreference.htm

Reply:
Reused of existing AssemblyInstruction which is "IfcExternalReference. Commit : 0bb5ae0d772bc5a99c0bea0886d3f8ed41fd2f5d